bonito/crf/model.py

```diff
@@ -176,6 +176,7 @@ def decode(self, x):
     def loss(self, scores, targets, target_lengths, **kwargs):
         return self.seqdist.ctc_loss(scores.to(torch.float32), targets, target_lengths, **kwargs)
 
+
 class Model(SeqdistModel):
```

Comment:
Please do imports in this way. Otherwise there is an error when import without CUDA:

```python
try:
    import seqdist.sparse
    from seqdist.ctc_simple import logZ_cupy, viterbi_alignments
except ImportError:
    pass
```

Reply:
We have another branch in progress @dkurt where we have migrated `seqdist` from `cupy` to precompiled kernels with a `cffi` interface so we should be able to include `cuda` and `openvino` options in a single package.

Reply:
The runtime dependency on CUDA is now avoided with the switch to `ont-koi`.